### PR TITLE
Add docs:

### PR DESCRIPTION
**Pull request by SLRover**
- jira module docs

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Atlassian Python API'
+copyright = '2018, SLRover'
```
Comment (gonchik) [inline docs/conf.py]: Hi! 
Thanks for awesome docs provides.
Can we set the Apache 2 license info? 
@AstroMatt  When we migrate into project, can we set the organisation copiright? 

Cheers,
Gonchik Tsymzhitov

File: docs/conf.py [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+# for a list of supported languages.
+#
+# This is also used if you do content translation via gettext catalogs.
+# Usually you set "language" from the command line for these cases.
+language = None
+
+# List of patterns, relative to source directory, that match files and
+# directories to ignore when looking for source files.
+# This pattern also affects html_static_path and html_extra_path.
+exclude_patterns = ['_build', 'Thumbs.db', '.DS_Store']
```
Comment (gonchik) [inline docs/conf.py]: Is it okay input info from .gitignore?

Comment (SLRover) [reply docs/conf.py]: @gonchik This patterns only for sphinx-build. I'll can add this to .gitignore.

Comment (gonchik) [reply docs/conf.py]: Awesome, thanks!

File: docs/conf.py (hunk 1 above)
Comment (AstroMatt) [reply docs/conf.py]: Yes. Let's migrate to the organization!
https://github.com/atlassian-api

This will be a new place for a project.

**Comment by SLRover** [comment]
[View docs test page](https://atlassian-python-api.readthedocs.io)

**Pull request merged**